### PR TITLE
Gp 1.1 annotation

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

**Review by jbech-linaro — COMMENTED** [review]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jenswi-linaro** [comment]
Tag applied

File: ta/crypt/cryp_taf.c [hunk 1]
```diff
@@ -609,35 +617,66 @@ TEE_Result ta_entry_ae_update(uint32_t param_type, TEE_Param params[4])
...
-	return TEE_AEDecryptFinal(op,
-			params[1].memref.buffer, params[1].memref.size,
-			params[2].memref.buffer, &params[2].memref.size,
-			params[3].memref.buffer, params[3].memref.size);
+	b2 = TEE_Malloc(params[2].memref.size, 0);
+	b3 = TEE_Malloc(params[3].memref.size, 0);
+	if (!b2 || !b3)
+		goto out;
+
+
```
Comment (etienne-lms) [inline ta/crypt/cryp_taf.c:667]: remove an empty line

**Review by etienne-lms — COMMENTED** [review]
commit "ta/crypt: use private buffers where needed":
in commit log s/require/requires/
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Review by etienne-lms — COMMENTED** [review]
commit "ta/storage: use private buffers where needed":
in commit log: s/require/requires/
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jenswi-linaro** [comment]
Comments addressed and tag applied

**Comment by jenswi-linaro** [comment]
We should probably merge this now that OP-TEE/optee_os#4080 was merged.

**Comment by jforissier** [comment]
@jenswi-linaro yes, thanks for the heads up :)

**Pull request merged**